tests/python/quantization/test_quantization.py

```diff
@@ -244,7 +245,8 @@ def check_quantized_pooling(data_shape, kernel, pool_type, pad, stride, global_p
         quantized_pooling = mx.sym.contrib.quantized_pooling(data=qdata, min_data=min_data,
                                                                 max_data=max_data, kernel=kernel,
                                                                 pad=pad, stride=stride, pool_type=pool_type,
-                                                                global_pool=global_pool)
+                                                                global_pool=global_pool,
+                                                                pooling_convention=convention)
```

Comment:
same question here: what was the default value for pooling_convention? Make sure the current case is covered.

Reply:
Should be "valid". https://mxnet.incubator.apache.org/api/python/ndarray/contrib.html#mxnet.ndarray.contrib.quantized_pooling